crates/bevy_input/src/gamepad.rs

```diff
@@ -243,6 +261,14 @@ impl ButtonAxisSettings {
     }
 }
 
+fn scale(value: f32, low: f32, high: f32) -> f32 {
```

Comment:
```suggestion
#[inline(always)]
fn scale(value: f32, low: f32, high: f32) -> f32 {
```

This helps ensure that the method can be optimized by the compiler.